scripts/lock.py

```diff
@@ -158,5 +158,10 @@ def parse_args():
         default=None,
         help='OS (distro) version such as "12.10"',
     )
+    parser.add_argument(
+        '--json-query',
+        default=None,
+        help='JSON frag/file of query parms for --list or --brief',
```

Comment:
I meant in the help text, so users may discover how to use this (pretty cool) feature.